### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by leo-ar**
- Recommend libraries to include in users' project configuration.
- Update links to the new typescript docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +38,10 @@ declare global {
 ```
 
 Now `myKVNamespace` is available to all of your source files.
+
+
+[tsconfig-lib]: https://www.typescriptlang.org/tsconfig#lib
+[`tsconfig.json`]: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html
+[`Request`]: https://developers.cloudflare.com/workers/reference/apis/request/
+[`Response`]: https://developers.cloudflare.com/workers/reference/apis/response/
+[`Headers`]: https://developer.mozilla.org/en-US/docs/Web/API/Headers
```
Comment (jrf0110) [inline README.md:47]: This is cool. I didn't know you could do this.

**Comment by jrf0110** [comment]
Thanks again @leo-ar !

**Comment by ispivey** [comment]
@leo-ar does including all of these libraries at the same time work for you? I ran into type definition conflicts when doing this, described in https://github.com/cloudflare/workers-types/issues/25

**Comment by leo-ar** [comment]
@ispivey our `tsconfig.json` has
```json
    "target": "ES6",
    "moduleResolution": "node",
    "allowSyntheticDefaultImports": true,
    "allowJs": true,
    "jsx": "react",
    "checkJs": true,
    "strict": true,
    "lib": ["DOM", "DOM.Iterable", "ES2018", "WebWorker"],
```
and it is type-checking correctly with `"typescript": "^3.9.2"`.
Since this worked and it is what our project needs I didn't try any other variations.

If you end up finding what the issue was, even if it was miss-configuration, consider adding it to the README.

**Comment by ispivey** [comment]
That's strange. Using your exact config, and `"@cloudflare/workers-types": "^1.0.9"`, and a nearly-trivial Worker script like the below, I get 39 errors caused by conflicts between lib.webworker.d.ts and lib.dom.d.ts upon transpilation.

```typescript
addEventListener('fetch', event => {
  event.respondWith(fetch('https://httpbin.org/get'))
})

export {}
```

We made some changes in #25 to address this, but I'd love to file a follow-up and figure out why that's working in your project, but not others.

**Comment by leo-ar** [comment]
@ispivey I was incorrect. I am seeing the errors you describe with our configuration too.

**Pull request merged**